.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron electron-mocha
+  - npm install
 node_js:
   - "6"
 notifications:
   email: false
+before_script:
+  - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start
+script:
+  - npm test
```

Comment:
Maybe we should name this `npm run test-node` in for symmetric purposes? If so, we can also make `npm test` run both of them.